configuration/blockly/index.md

```diff
@@ -51,13 +51,43 @@ Using this library you can not only create much simpler code, it also allows **n
 
 ### openHAB 3 / openHAB 4 - Migration
 
-This has several implications:
+#### Migrating Blockly rules to openHAB 4 is easy
 
-- From openHAB 4 on, the default script engine is GraalJS when Blockly creates new scripts
-- If you still want to run Blockly rules that were created in openHAB3 without changing them (see below) you can install an automation plugin for Nashorn which could provide backwards compatibility until you have converted all rules.
-After installation both old code and new code can run in openHAB4.
-- **To convert a rule that was created in openHAB 3 (NashornJS) to a new rule based on GraalJS for openHAB 4 simply open the rule once in openHAB 4 and save it - that's it.**
-- After all rules have been converted, please uninstall the Nashorn addon to save memory.
+From openHAB 4 on, the default script engine is GraalJS when Blockly creates new scripts.
+From a technical perspective a rule internally holds a so-called MIME-type that tells openHAB how the generated JavaScript language has to be interpreted.
+The (default) MIME-type application/javascript in openHAB 3 runs the rule with NashornJS, while this same MIME-type will run the Blocky rule with GraalJS.
```

Comment:
```suggestion
The (default) MIME-type `application/javascript` in openHAB 3 runs the rule with NashornJS, while this same MIME-type will run the Blocky rule with GraalJS in openHAB 4.
```

Reply:
To me and the user, these ARE rules. I would rather keep it that way here.

Reply:
You can create scripts in the `Script` tab of UI.
And: In real rules (I mean inside the Rules tab), you add either an action called `Run Script` or `execute a given script` to run something with Blockly.
So IMO these are scripts for the users.